picoquic/sender.c

```diff
@@ -2225,6 +2227,7 @@ int picoquic_prepare_packet_ready(picoquic_cnx_t* cnx, picoquic_path_t * path_x,
 
         if (ret == 0 && retransmit_possible &&
             (length = picoquic_retransmit_needed(cnx, pc, path_x, current_time, packet, send_buffer_min_max, &is_cleartext_mode, &header_length)) > 0) {
+            packet->contains_retrans = 1; 
             /* Set the new checksum length */
```

Comment:
Please don't do that. Instead, you may want to call something like "picoquic_loss_detected(cnx, pc, path_x)" that will start some accounting in your "spin bit QR" context. (Note that there is a separate context for each path).

Reply:
Also, that call should be inside the `picoquic_retransmit_needed()` function, not in the calling function.